06-building-a-program-start-to-finish/solving-the-problem/output/invertedIndex.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * An inverted-index class
+ * @class
+ */
+class InvertedIndex {
+  /**
+  * class constructor
+  * @constructor
+  */
+  constructor() {
+    this.index = {};
+    this.object = {};
+    this.allFiles = {};
+    this.allLength = {};
+    this.allFilesTitle = [];
+  }
+  /**
+   * Gets Indexes
```

Comment:
Bayo thank you for that observation, I created a get Indicies method to get indicies